generic_expression_patterns_modules/make_GiG_network.R

```diff
@@ -0,0 +1,41 @@
+# Using R 3.6.3
+
+# Manually load operons.rda and geneinfo.rda into R environment
+# from https://github.com/greenelab/ADAGEpath/blob/master/data/
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager")
+BiocManager::install("impute")
+BiocManager::install("greenelab/ADAGEpath")
+
+# Get eADAGE model
+eADAGEmodel <- ADAGEpath::eADAGEmodel
+
+# Get all eADAGE signatures

```

Comment:
I'm not too familiar with how eADAGE/ADAGEpath plots networks. Are you only generating a network for certain genes (or certain pathways/gene signatures), or is this all Pseudomonas genes/interactions? 

What do the edges represent here: pathway relationships? Physical interactions between genes/proteins?

I know a lot of these questions are probably answered in the ADAGEpath repo, but it might be good to give a short synopsis here.

Reply:
This is a very good point. I will add this detail to the script. Essentially the networks were generated by using weights generated by eADAGE (denoising autoencoder) where nodes = genes and edges = similar weight profiles for how much that gene contributes to the eADAGE LV. This network is showing all genes, not a subset. The clustering is using pearson correlation with a default cutoff of 0.5